_includes/v21.2/sql/generated/diagrams/alter_role.html

```diff
@@ -1,29 +1,102 @@
-<div><svg width="751" height="81">
-<polygon points="9 17 1 13 1 21"></polygon>
-<polygon points="17 17 9 13 9 21"></polygon>
-<rect x="31" y="3" width="62" height="32" rx="10"></rect>
-<rect x="29" y="1" width="62" height="32" class="terminal" rx="10"></rect>
-<text class="terminal" x="39" y="21">ALTER</text>
-<rect x="133" y="3" width="56" height="32" rx="10"></rect>
-<rect x="131" y="1" width="56" height="32" class="terminal" rx="10"></rect>
-<text class="terminal" x="141" y="21">ROLE</text>
-<rect x="133" y="47" width="56" height="32" rx="10"></rect>
-<rect x="131" y="45" width="56" height="32" class="terminal" rx="10"></rect>
-<text class="terminal" x="141" y="65">USER</text>
-<rect x="249" y="35" width="34" height="32" rx="10"></rect>
-<rect x="247" y="33" width="34" height="32" class="terminal" rx="10"></rect>
-<text class="terminal" x="257" y="53">IF</text>
-<rect x="303" y="35" width="70" height="32" rx="10"></rect>
-<rect x="301" y="33" width="70" height="32" class="terminal" rx="10"></rect>
-<text class="terminal" x="311" y="53">EXISTS</text>
-<rect x="413" y="3" width="56" height="32"></rect>
-<rect x="411" y="1" width="56" height="32" class="nonterminal"></rect>
-<text class="nonterminal" x="421" y="21">name</text><a xlink:href="sql-grammar.html#opt_with" xlink:title="opt_with">
-<rect x="509" y="35" width="74" height="32"></rect>
-<rect x="507" y="33" width="74" height="32" class="nonterminal"></rect>
-<text class="nonterminal" x="517" y="53">opt_with</text></a><a xlink:href="sql-grammar.html#role_options" xlink:title="role_options">
-<rect x="603" y="35" width="100" height="32"></rect>
-<rect x="601" y="33" width="100" height="32" class="nonterminal"></rect>
-<text class="nonterminal" x="611" y="53">role_options</text></a><path class="line" d="m17 17 h2 m0 0 h10 m62 0 h10 m20 0 h10 m56 0 h10 m-96 0 h20 m76 0 h20 m-116 0 q10 0 10 10 m96 0 q0 -10 10 -10 m-106 10 v24 m96 0 v-24 m-96 24 q0 10 10 10 m76 0 q10 0 10 -10 m-86 10 h10 m56 0 h10 m40 -44 h10 m0 0 h134 m-164 0 h20 m144 0 h20 m-184 0 q10 0 10 10 m164 0 q0 -10 10 -10 m-174 10 v12 m164 0 v-12 m-164 12 q0 10 10 10 m144 0 q10 0 10 -10 m-154 10 h10 m34 0 h10 m0 0 h10 m70 0 h10 m20 -32 h10 m56 0 h10 m20 0 h10 m0 0 h204 m-234 0 h20 m214 0 h20 m-254 0 q10 0 10 10 m234 0 q0 -10 10 -10 m-244 10 v12 m234 0 v-12 m-234 12 q0 10 10 10 m214 0 q10 0 10 -10 m-224 10 h10 m74 0 h10 m0 0 h10 m100 0 h10 m23 -32 h-3"></path>
-<polygon points="741 17 749 13 749 21"></polygon>
-<polygon points="741 17 733 13 733 21"></polygon></svg></div>
+<div><svg width="1209" height="657">
```

Comment:
I assume we won't need this change after https://github.com/cockroachdb/docs/pull/11846 is merged?